snippets.yaml

```diff
@@ -0,0 +1,68 @@
+Dataframes.Table:
+  $global:
+    - name: read "path/to/data"
```

Comment:
As we already discussed for stdlib snippets, this should be made into an actual readable path when we can.

Reply:
Also, it should be `Table.read` if it's a global snippet.

Reply:
https://github.com/luna/dataframes/issues/142